redaxo/src/addons/users/lang/de_de.lang

```diff
@@ -27,7 +27,9 @@ user_extras = Extras
 last_login = Letzter Login
 user_caption = Liste der angelegten Benutzer
 user_password_change_required = Passwort muss nach Login ausgetauscht werden
-user_status = User ist aktiv
+user_status = Status
```

Comment:
Scheinbar wird dieses Feld dann nicht mehr verwendet. Ich konnte zumindest keine andere Stelle finden.
Dann würde ich es ganz entfernen (am besten in allen lang files).

Reply:
Erledigt.